src/nti/transactions/transactions.py

```diff
@@ -237,6 +253,16 @@ def do(*args, **kwargs):
     transaction.get().join(result)
     return result
 
+def do_last(*args, **kwargs):
```

Comment:
Only if you read the docstrings. That doesn't help someone just reading the code where this is used.